src/index.js

```diff
@@ -0,0 +1,45 @@
+import './styles.css';
+import addList from './addToList.js';
+
+const todoList = document.querySelector('#todoList');
+const inputFaild = document.querySelector('#inputFaild');
+// const clearAll = document.querySelector('#clearAll');
+
+const stock = [];
```

Comment:
- [ ] The project should contain an array of some simple to do tasks (array of objects) with each object at least containing three keys - description [string], completed [bool], index: [number]. Kindly check the project requirements.
- [ ] Kindly make sure that a list of some tasks is rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task.

Reply:
- [ ] good job addressing some of the suggestions made by the previous CR 👍 however what the previous CR is referring to is the below requirements:
- Create an **index.js** file and set an array of some simple *to do tasks* (array of objects). Each task object should contain three keys: 
  1. description [`string`].
  2. completed [`bool`].
  3. index: [`number`].

### This means that you are required to populate the fields as well with a few placeholder projects.